docs/fileformat-prose.md

```diff
@@ -10,21 +10,42 @@ AnnData objects are saved on disk to hierarchical array stores like [HDF5]
 (via {doc}`H5py <h5py:index>`) and {doc}`zarr:index`.
 This allows us to have very similar structures in disk and on memory.
 
-As an example we’ll look into a typical `.h5ad` object that’s been through an analysis.
```

Comment:
I'd add a link to download the dataset here as it's not immediate how to actually access the dataset being considered.

Reply:
this both for the .h5ad and the .zarr file

Reply:
I'm going to leave this to a follow up. I'd rather get this in, than wait on figuring out hosting + making a smaller dataset (this one is like 6gb).